### PR TITLE
Add Solo and Symphony

### PR DESCRIPTION
**Pull request by 88250**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
   * [Noddity](http://noddity.com/) - It's a blog, it's a wiki, it's a fast CMS. ([Source Code](https://github.com/TehShrike/noddity)) `WTFPL` `Nodejs`
   * [PluXml](http://www.pluxml.org/) - XML-based blog/CMS platform. ([Source Code](https://github.com/pluxml/PluXml)) `GPL` `PHP`
   * [Postleaf](https://www.postleaf.org/) - An open source blogging platform with inline editing, handlebar templates, and a beautiful user interface. ([Source Code](https://github.com/Postleaf/postleaf)) `GPLv3` `PHP`
+  * [Solo](https://github.com/b3log/solo) - A blogging system written in Java, feel free to create your or your team own blog. ([Demo](http://88250.b3log.org/), [Source Code](https://github.com/b3log/solo)) `Apache` `Java`
```
Comment (AndyR207) [inline README.md]: Please remove the Source Code link as it is the same as the homepage URL and duplicate links are not allowed.

      * [Solo](https://github.com/b3log/solo) - A blogging system written in Java, feel free to create your or your team own blog. ([Demo](http://88250.b3log.org/)) `Apache` `Java`

File: README.md [hunk 2]
```diff
@@ -362,6 +363,7 @@ _[IPBX](https://en.wikipedia.org/wiki/IP_PBX) telephony software_
   * [Pump.io](http://pump.io/) - Stream server that does most of what people really want from a social network. ([Source Code](https://github.com/e14n/pump.io)) `Apache` `Nodejs`
   * [Reddit](https://www.reddit.com/) - Social networking and news website where registered community members can submit content. ([Demo](https://www.reddit.com/), [Source Code](https://github.com/reddit/reddit)) `CPALv1` `Python`
   * [RedMatrix](https://github.com/redmatrix/redmatrix) - Complete decentralised publishing, sharing, and communications system. `MIT` `PHP`
+  * [Symphony](https://github.com/b3log/symphony) - A modern community (forum/SNS/blog) platform written in Java. ([Demo](https://hacpai.com), [Source Code](https://github.com/b3log/symphony)) `GPLv3` `Java`
```
Comment (AndyR207) [inline README.md]: Please remove the Source Code link as it is the same as the homepage URL and duplicate links are not allowed.

     * [Symphony](https://github.com/b3log/symphony) - A modern community (forum/SNS/blog) platform written in Java. ([Demo](https://hacpai.com)) `GPLv3` `Java`

File: README.md (hunk 1 above)
Comment (88250) [inline README.md]: Thank you for review, I updated just now.

File: README.md (hunk 2 above)
Comment (88250) [inline README.md]: Thank you for review, I updated just now.

**Pull request merged**